### ISSUE TITLE
Hugo fails to build after new incident is created

### ISSUE THREAD

**Issue by steveoh** [netlify/netlify-statuskit#25]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
nil error on Incident template

**- If the current behavior is a bug, please provide the steps to reproduce.**
1. clone repo
1. npm install
1. cd site
1. hugo new incidents/fail.md
1. update to resovled
1. npm start
1. fail

**- What is the expected behavior?**
it builds

**- Please mention your node.js, and operating system version.**

This is a hugo issue I made
https://discourse.gohugo.io/t/netlify-status-kit-upgrade-issue/8175

**Comment by bep** [comment]
The main reason this fails in newer versions of Hugo is that `.Site.Pages` now returns all types of pages, including home page etc.

So:

```
{{ $incidents := where .Site.Pages.ByDate.Reverse "Section" "incidents" }}
```

Could be changed to:

```
{{ $incidents := where .Site.RegularPages.ByDate.Reverse "Section" "incidents" }}
```

But you should also protect against missing page params, something ala:

```html
  {{- $severity := .Params.severity  | default "under-maintenance" }}
  {{- $alert := index .Site.Data.severity.alerts $severity -}}

  <p class="incident-status color-{{ $alert }}">
    {{ index .Site.Data.severity.descriptions $severity }}
```

But note that I doubt that any of the above is backwards compatible with Hugo 0.17 (not sure about `default`), which falls heavily into the "no longer supported" category.

**Comment by verythorough** [comment]
Thanks for the tip, @bep, and thanks @steveoh for the fix! 

Closed by #26
